core/modules/sql/SqlTransactionStd.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * LSST Data Management System
+ * Copyright 2015 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_SQL_SQLTRANSACTIONNORMAL_H
+#define LSST_QSERV_SQL_SQLTRANSACTIONNORMAL_H
+
+// System headers
+
+// Third-party headers
+
+// Qserv headers
+#include "sql/SqlTransaction.h"
+#include "util/Issue.h"
+
+namespace lsst {
+namespace qserv {
+namespace sql {
+
+/// A standard way to use SqlTransaction.
```

Comment:
I would probably rephrased this and renamed the class. The class implements an RAII idiom for transaction management, it may be "standard" but most people are probably used more to the names like "Guard" or "Scope".